db/migrate/20230314202847_update_bad_donation_issued_ats.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateBadDonationIssuedAts < ActiveRecord::Migration[7.0]
+  Donation.unscoped.where("issued_at <  ?","2000-01-01").find_each do |donation|
+    donation.update_attribute(:issued_at, donation.created_at)
```

Comment:
This is just the same migration twice?

Reply:
Nope - one's distribution, one's donation.  We don't need one for purchases.